src/effects/VST3/internal/ParameterChanges.h

```diff
@@ -0,0 +1,50 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file ParameterChanges.h
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#pragma once
+
+#include <vector>
+
+#include <pluginterfaces/vst/ivstparameterchanges.h>
+#include "ParameterValueQueue.h"
+#include "ParameterQueuePool.h"
+
+namespace internal
+{
+   /** Implements IParameterChanges, uses ParameterQueuePool internally
+     * to avoid unnecessary allocations. Though both getParameterData
```

Comment:
This comment block does not separate brief and detailed descriptions.  So all will appear only in the detailed description on the Doxygen pages.

This isn't wrong, but you might reconsider what would best help someone else using those pages.

Reply:
For that purpose it would be better to read `IParameterChanges` documentation, I cannot imagine anything helpful I could add that isn't a duplicate of what's written there